### ISSUE TITLE
Does it work with systemd as pid 1?

### ISSUE THREAD

**Issue by adrianog** [scottyhardy/docker-remote-desktop#34]

**Comment by scottyhardy** [comment]
No, I don't use systemd as it's not recommended for container environments - instead the entrypoint runs the required services directly.  Alternatively I've seen linuxserver/docker-rdesktop use s6-overlay as a lightweight alternative but is far too complicated to configure and I just want to keep my image as simple as possible.

**Comment by adrianog** [comment]
I came across the requirement of systemd when trying to use snap for installations.

**Comment by scottyhardy** [comment]
Ah, I’ve just realised you would have been using the old version of docker-wine that was based on Ubuntu 20.04 focal, which was still using sysvinit/upstart. Systemd is installed with the latest Ubuntu 24.04 docker-wine images as it’s a pre-requisite for xorgxrdp and others.  I don’t actually run systemd, but installs requiring it all seem to work ok albeit with some warnings. No issues with snap installs I’ve tried either.

Let me know if you’re still having problems with the latest container images.
